addons/download-button/addon.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Project page download button",
+  "description": "Allows you to download projects directly from the project page, without having to open the editor first.",
```

Comment:
Would be nice to say where the button is located.
`next to the Copy Link button`?